src/pages/home/report/ReportActionCompose.js

```diff
@@ -1050,121 +1040,119 @@ class ReportActionCompose extends React.Component {
-                                                    )}
-                                                    <Tooltip text={this.props.translate('reportActionCompose.addAction')}>
-                                                        <PressableWithFeedback
-                                                            ref={this.actionButtonRef}
-                                                            onPress={(e) => {
-                                                                e.preventDefault();
-
-                                                                // Drop focus to avoid blue focus ring.
-                                                                this.actionButtonRef.current.blur();
-                                                                this.setMenuVisibility(!this.state.isMenuVisible);
-                                                            }}
-                                                            style={styles.composerSizeButton}
-                                                            disabled={isBlockedFromConcierge || this.props.disabled}
-                                                            accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
-                                                            accessibilityLabel={this.props.translate('reportActionCompose.addAction')}
-                                                        >
-                                                            <Icon src={Expensicons.Plus} />
-                                                        </PressableWithFeedback>
-                                                    </Tooltip>
-                                                </View>
-                                                <PopoverMenu
-                                                    animationInTiming={CONST.ANIMATION_IN_TIMING}
-                                                    isVisible={this.state.isMenuVisible}
-                                                    onClose={() => this.setMenuVisibility(false)}
-                                                    onItemSelected={(item, index) => {
-                                                        this.setMenuVisibility(false);
-
-                                                        // In order for the file picker to open dynamically, the click
-                                                        // function must be called from within a event handler that was initiated
-                                                        // by the user.
-                                                        if (index === menuItems.length - 1) {
-                                                            // Set a flag to block suggestion calculation until we're finished using the file picker,
-                                                            // which will stop any flickering as the file picker opens on non-native devices.
```

Comment:
One minor update - can we copy this comment over to new L1045 so we keep the note?